README.md

```diff
@@ -14,14 +14,10 @@ the process of aligning security controls from SRG items into actual STIG securi
 * Confidential data in the database is encrypted using symmetric encryption
 * Authenticate via the local server, through github, and through configuring an LDAP server.
 
-## Installation
-
-### Dependencies
-
-For Docker:
-  * Docker
-  * docker-compose
+## Configuration Breakdown
```

Comment:
Maybe just make this called "Deploy Vulcan" instead of Configuration Breakdown and then just link to the Installation Instructions from here to make it more clear that this where to go if you're trying to deploy the application.